composer.lock

```diff
@@ -4,7 +4,7 @@
         "Read more about it at https://getcomposer.org/doc/01-basic-usage.md#installing-dependencies",
         "This file is @generated automatically"
     ],
-    "content-hash": "93367a3be337e7b590e4ba9417978c79",
+    "content-hash": "6286d4a3985d893a63324ad5686e38e9",
```

Comment:
What is this file?

Reply:
check the composer.json file. I modified it to add the pusher library. So the compose.lock file updated its hash